src/Store/Features.re

```diff
@@ -143,6 +143,29 @@ let update =
       };
     (state, effect);
 
+  | Layout(msg) =>
+    open Feature_Layout;
+
+    let focus =
+      EditorGroups.getActiveEditorGroup(state.editorGroups)
+      |> Option.map((group: EditorGroup.t) => group.editorGroupId);
+    let (model, maybeOutmsg) = update(~focus, state.layout, msg);
+    let state = {...state, layout: model};
+
+    let state =
+      switch (maybeOutmsg) {
+      | Focus(editorGroupId) => {
+          ...state,
+          editorGroups:
+            EditorGroups.setActiveEditorGroup(
+              editorGroupId,
+              state.editorGroups,
+            ),
+        }
+      | Nothing => state
+      };
+    (state, Effect.none);
+
```

Comment:
The active editor group is now managed by the layout feature, but still owned at root level and just passed into the update and view. We could probably move it into the `Focus/FocusManager` mechanism too.